src/clojure/lazytest/extensions/expectations.cljc

```diff
@@ -363,7 +363,7 @@
   "Given a name (a symbol that may include metadata) and a test body,
   produce a standard `lazytest.core` test var (using `defdescribe`)."
   [n & body]
-  (with-meta `(lt/defdescribe ~n ~@body) (meta &form)))
+  (with-meta `(lt/defdescribe ~n (lt/it ~(str n) ~@body)) (meta &form)))
```

Comment:
I've used the test name symbol as the docstring here. I think this produces nicer output than the `deftest` equivalent which passes `nil` in this context -- maybe change `deftest` to pass the test name symbol as the docstring too?